xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1241,6 +1243,22 @@ bool CXBMCApp::WaitVSync(unsigned int milliSeconds)
   return m_vsyncEvent.WaitMSec(milliSeconds);
 }
 
+bool  CXBMCApp::GetMemoryInfo(long &availMem, long &totalMem)
```

Comment:
It seems you forgot to assign `availMem` and `totalMem`? And extra space

Reply:
left align `&`

Reply:
@garbear assignement is done in line 1258 / 1259